css/properties.json

```diff
@@ -4772,6 +4743,52 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset-distance": {
+    "syntax": "&lt;length-percentage&gt;",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "lpc",
+    "percentages": "referToTotalPathLength",
+    "groups": [
+      "CSS Motion"
+    ],
+    "initial": "<code>0</code>",
+    "appliesto": "allElementsSVGContainerElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
+    "status": "experimental"
+  },
+  "offset-path": {
+    "syntax": "&lt;url&gt; | [ &lt;basic-shape&gt; | &lt;path()&gt; ] || &lt;geometry-box&gt; | none",
```

Comment:
The syntax of `offset-path` [got changed to](https://drafts.fxtf.org/motion-1/#propdef-offset-path):

    none | ray( [ <angle> && <size>? && contain? ] ) | <path()> | <url> | [ <basic-shape> || <geometry-box> ]

Sebastian